src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -22,8 +21,9 @@ const MenuIcon = styled(EllipsisV)`
   user-select: none;
 `;
 
+/*this should be a button but Safari and firefox don't support focusing buttons*/
```

Comment:
lets add some links to this comment 

https://bugs.webkit.org/show_bug.cgi?id=22261
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus